src/utils/mixins.js

```diff
@@ -71,3 +73,25 @@ export const vueCrudMixins = {
         }
     }
 }
+
+export const vuexMixins = {
+    computed: {
+        ...mapGetters("Company", ["currentCompanyID"])
```

Comment:
Since there is a request to use camelCase variable this one needs to be updated.
```suggestion
        ...mapGetters("Company", ["currentCompanyId"])
```